src/main/java/io/codelex/advancedtest/exercise1/CreditCard.java

```diff
@@ -0,0 +1,31 @@
+package io.codelex.advancedtest.exercise1;
+
+import java.math.BigDecimal;
+
+public class CreditCard extends Card{
+
+    private BigDecimal balance;
```

Comment:
Šis mainīgais varēja būt Card klasē, jo kā redzi - viņš ir gan CreditCard, gan DebitCard